src/main/java/com/kodilla/ecommercee/exception/ProductNotFoundException.java

```diff
@@ -0,0 +1,11 @@
+package com.kodilla.ecommercee.exception;
+
+import org.springframework.http.HttpStatus;
+import org.springframework.web.server.ResponseStatusException;
+
+public class ProductNotFoundException  extends ResponseStatusException {
+
+    public ProductNotFoundException() {
```

Comment:
Wyjątek na razie nie użyty, nie wiem czy to nie powinno być później tworzone.

Reply:
Utworzone by nie zapomnieć później, poza tym IDE czasem podpowiada by rzucić wyjątkiem więc nie polemizuje z nim.